docs/docs/userguide/upgrades/HELM_CHART_UPGRADE.md

```diff
@@ -0,0 +1,278 @@
+# Helm chart upgrade
+Each Helm chart has a default product version that might change in next Helm chart 
+version. So be aware that if you upgrade the Helm chart, it might lead to upgrading the product as well. This depends on the current and target Helm chart versions. 
+
+!!!important "Do you want to upgrade the product to a new version?" 
+     If you want to upgrade the product version without upgrading the Helm chart then 
+     refer to [Product upgrades](PRODUCTS_UPGRADE.md). 
+
+Before upgrading the Helm chart, first consider: 
+* the version of the current Helm chart
+* the version of the product running on your Kubernetes cluster
+* the target version of the Helm chart you want to upgrade to
+* the target version of the product you want to upgrade to
```

Comment:
These bullet points dont render properly without a space after `Before upgrading the Helm chart, first consider: `
```suggestion
Before upgrading the Helm chart, first consider: 

* the version of the current Helm chart
* the version of the product running on your Kubernetes cluster
* the target version of the Helm chart you want to upgrade to
* the target version of the product you want to upgrade to
```